components/prism-jolie.js

```diff
@@ -0,0 +1,47 @@
+Prism.languages.jolie = Prism.languages.extend('clike', {
+	'keyword': /\b(?:include|define|is_defined|undef|main|init|outputPort|inputPort|Location|Protocol|Interfaces|RequestResponse|OneWay|type|interface|extender|throws|cset|csets|forward|Aggregates|Redirects|embedded|courier|extender|execution|sequential|concurrent|single|scope|install|throw|comp|cH|default|global|linkIn|linkOut|synchronized|this|new|for|if|else|while|in|Jolie|Java|Javascript|nullProcess|spawn|constants|with|provide|until|exit|foreach|instanceof|over|service)\b/g,
+	'builtin': /\b(?:undefined|string|int|void|long|Byte|bool|double|float|char|any)\b/,
+	'number': /\b(?:\d*\.?\d+(e[+-]?\d+)?l?)\b/i,
+	'operator': /->|<<|=<|=>|[!+->=]?=|[:<>!?*\/%^]|&&|\|\||--?|\+\+?/g,
```

Comment:
I can't see any support for `<=` here. Supposing it exists, it probably should be added to `[!+->=]?=`.
On the opposite, I couldn't find any reference to the `=<` operator from a quick look at the docs. Please confirm this isn't a typo.
I think Jolie also supports `*=` and `%=` (but not `/=`... ?). Could you check?